app/models/site.rb

```diff
@@ -18,4 +18,40 @@ def instance
       first_or_create
     end
   end
+
+  # Get all administrator emails associated with this site
+  def admin_emails
+    User.with_role(:admin, self).pluck(:email)
+  end
+
+  # Update administrator emails associated with this site
+  # @param [Array<String>] Array of user emails
+  def admin_emails=(emails)
+    existing_admin_emails = admin_emails
+    new_admin_emails = emails - existing_admin_emails
+    removed_admin_emails = existing_admin_emails - emails
+    add_admins_by_email(new_admin_emails) if new_admin_emails
+    remove_admins_by_email(removed_admin_emails) if removed_admin_emails
+  end
+
+  private
+
```

Comment:
This method (below) appears to accept only one email, as in `@param [String] email` but the name of the param implies a plural array of emails.  Could the param name be singular and add an `@param` docstring to clarify the intended param?